hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MetricsPlugin.java

```diff
@@ -70,35 +70,38 @@ public long getPeriodMillis() {
 
     @Override
     public void run(DiagnosticsLogWriter writer) {
-        writer.startSection("Metrics");
         probeRenderer.writer = writer;
+        // we set the time explicitly so that for this particular rendering of the probes, all metrics have exactly
```

Comment:
Ah, thx. I misread `MetricsPlugin` as a specialized plugin, not a common class. Makes sense now.